examples/00_intro_to_thinc.ipynb

```diff
@@ -44,9 +44,9 @@
    },
    "outputs": [],
    "source": [
-    "import thinc.util\n",
+    "import thinc.compat\n",
```

Comment:
This should remain backwards-compatible, so let's (make sure that's the case and) undo these changes to the examples.